pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseSingleStageBrokerRequestHandler.java

```diff
@@ -448,6 +448,14 @@ protected BrokerResponse handleRequest(long requestId, String query, @Nullable S
       }
 
       // Validate QPS quota
+      String database = DatabaseUtils.extractDatabaseFromTableName(tableName);
+      if (!_queryQuotaManager.acquireDatabase(database)) {
+        String errorMessage =
+            String.format("Request %d: %s exceeds query quota for database: %s", requestId, query, database);
+        LOGGER.info(errorMessage);
```

Comment:
nit: can use parameterized logging instead

Reply:
Need the error message below for response as well hence formulating it first.

Reply:
Whoops, missed that, my bad.